integration-tests/luks/create-with-ext4.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+
+# requirements: disk /dev/sdb with empty partition sdb1
+
+
+from sys import exit
+from storage import *
+from storageitu import *
+
+
+set_logger(get_logfile_logger())
+
+environment = Environment(False)
+
+storage = Storage(environment)
+
+staging = storage.get_staging()
+
+sdb1 = Partition.find_by_name(staging, "/dev/sdb1")
+
+luks = sdb1.create_encryption("cr_test")
```

Comment:
"Create encryption" does not make much sense. Can we rename this to "Create encrypted device/volume/partition/whatever"?